Ferlab-Ste-Justine/clin-frontend
# fix: #233 remove some warnings fix probant display

# [233] [Display family position correctly]

closes #[233]

## Description
In the family tab, family position should be displayed correctlty.

Critères de succès

## Validation


## Screenshot

before
![beforeScreenshot](https://user-images.githubusercontent.com/54366437/133821390-ca8485fe-3dbd-4ac7-b885-07137cf8697a.png)

after
![afterscreenshot](https://user-images.githubusercontent.com/54366437/133821412-4eb833ad-0742-41da-ae8b-76cd0f0e74d6.png)



## QA

manually test in the family tab of patient page that family position is correctly displayed. If not, pls check the data in fhir so that we can see if the issue is on the backend or the frontend.
...
